app/main/helpers/email.py

```diff
@@ -49,5 +50,12 @@ def decode_email(token):
     ts = URLSafeTimedSerializer(main.config["SECRET_KEY"])
     decoded = ts.loads(token,
                        salt=main.config["RESET_PASSWORD_SALT"],
-                       max_age=86400)
+                       max_age=86400, return_timestamp=True)
     return decoded
+
+
+def token_created_before_password_last_changed(token_timestamp, user):
+
+        password_last_changed = datetime.strptime(
+            user['users']['passwordChangedAt'], '%a, %d %b %Y %H:%M:%S %Z')
```

Comment:
So should we enforce the correct format in this app (with a fallback if we actually get the format we expect) so that we can merge it and then correct the output from the API?

Also, I've used that date format string in two places -- there's one in the tests as well.  Would it be better to store it in a config variable?